docs/reference/commandline/node_ls.md

```diff
@@ -161,6 +166,11 @@ e216jshn25ckzbvmwlnh5jr3g: swarm-manager1 Ready
 35o6tiywb700jesrt3dmllaza: swarm-worker1 Needs Rotation
 ```
 
+To print all the nodes with json output, use `json` directive:
+```bash
+$ docker node ls --format json
+{"Availability":"Active","EngineVersion":"20.10.5","Hostname":"docker-desktop","ID":"k8f4w7qtzpj5sqzclcqafw35g","ManagerStatus":"Leader","Self":true,"Status":"Ready","TLSStatus":"Ready"}
```

Comment:
Looks like we're (already) not consistent with pretty/non-pretty printing JSON 🤔